website/docs/reference/datasources/querying-postgres.md

```diff
@@ -3,78 +3,88 @@ sidebar_position: 13
 ---
 # PostgreSQL
 
-This document covers how to establish a connection between your PostgreSQL database and Appsmith to read and write data on your applications.
+This page gives information to connect Appsmith to a PostgreSQL database and to read and write data in your applications.
 
+## Connect PostgreSQL
 
-## Connect to PostgreSQL datasource
-
-To add a PostgreSQL datasource, navigate to **Explorer** >> click the (**+**) sign next to **Datasources** >> select PostgreSQL under Databases. This opens up the page where you can configure the parameters to connect to your PostgreSQL database.
-
-
-### Configure datasource
-
-To connect to a PostgreSQL database, you must configure the following parameters. All required fields are suffixed with an asterisk (\*).
+:::caution 
+If you are a self-hosted user, you need to configure the `pg_hba.conf` file to whitelist the IP addresses `18.223.74.85` and `3.131.104.27` of the Appsmith deployment on your database instance before connecting to a database. See [**Client Authentication**](https://www.postgresql.org/docs/current/auth-pg-hba-conf.html) for more details.
+:::
 
 <figure>
-  <img src="/img/postgres-img.png" style= {{width:"700px", height:"auto"}} alt="Configure PostgreSQL Datasource"/>
-  <figcaption align = "center"><i>Configure PostgreSQL Datasource</i></figcaption>
+  <img src="/img/postgres-img.png" style= {{width:"100%", height:"auto"}} alt="Connect PostgreSQL Database"/>
+  <figcaption align = "center"><i>Connect PostgreSQL Database</i></figcaption>
 </figure>
 
-* **Connection Mode\*:** This refers to the permission granted to Appsmith when establishing a connection to the database. The two available modes are:
+### Connection parameters
```

Comment:
The formats in MongoDB and PostgreSQL are different. Are these two types of datasource templates that we'll be looking at, and once finalized, will be using one of them?